Modules/xSharePoint/Modules/xSharePoint.Util/xSharePoint.Util.psm1

```diff
@@ -310,4 +310,15 @@ function Set-xSharePointObjectPropertyIfValueExists() {
     }
 }
 
+# Due to issues with unit testing, this function has been created so we are able to mock this function.
+function Remove-WebAppPolicy() {
```

Comment:
Really? You should be able to put it in to the util module and then when you mock specify the module name:

    Mock Remove-WebAppPolicy { } -ModuleName "xSharePoint.Util"

If you can't get it let me know and I'll take a look.